### PR TITLE
Feature/user

### PR DESCRIPTION
**Pull request by hugh-n**
コード修正

### CODE REVIEW

File: app/assets/stylesheets/layout/user/_footer.scss [hunk 1]
```diff
@@ -1,10 +1,29 @@
+.p-users-footer-container{
+  margin-right: -15px;
+  margin-left: -15px;
+  display: flex;
+  -webkit-box-pack: center;
+  justify-content: center;
+}
```
Comment (hugh-n) [inline app/assets/stylesheets/layout/user/_footer.scss]: CSS ordering must be changed to A to Z

File: app/assets/stylesheets/layout/user/_footer.scss [hunk 2]
```diff
@@ -1,10 +1,29 @@
...
+
+  &__title{
+    color: #FFFFFF;
+    font-size: 16px;
+    height: 30px;
+    margin-top: 40px;
+    padding: 10px;
+    width: 300px;
+    text-decoration: underline;
+  }
```
Comment (hugh-n) [inline app/assets/stylesheets/layout/user/_footer.scss]: CSS ordering must be changed to A to Z

File: app/assets/stylesheets/layout/user/_footer.scss [hunk 3]
```diff
@@ -1,18 +1,49 @@
...
+  color: #FFFFFF;
+  height: auto;
+  left: 0;
   position: absolute;
   right: 0;
-  bottom: 0;
-  left: 0;
+  width: 100%;
+  padding-bottom: 30px;
+
```
Comment (hugh-n) [inline app/assets/stylesheets/layout/user/_footer.scss]: CSS ordering must be changed to A to Z

File: app/controllers/users/service_controller.rb [hunk 4]
```diff
@@ -1,14 +1,17 @@
 class Users::ServiceController < ApplicationController
+	PER = 20
 	def index
 		@check = MServiceCategory.find_by(:name_alias => params["category_id"])
```
Comment (ericgpks) [inline app/controllers/users/service_controller.rb]: find_by(name_alias: params[:category_id])

File: app/views/elements/service/_businessdata.html.haml [hunk 5]
```diff
@@ -25,12 +25,10 @@
...
-            .table-striped.p-i-service__table-tag
-              - getBusinessService(business.id).each do |service|
-                .p-i-service__table-tag-detail
-                  %i.fa.fa-tag
-                  = getService(service.service_id)
             .table-striped.p-i-service__table-des
               = business.explanation
             .table-striped.p-i-service__table-btn
               = link_to "詳細を見る", "/admin/businesses/", class: "btn p-i-service__green"
+    = if @count > 20
```
Comment (ericgpks) [inline app/views/elements/service/_businessdata.html.haml]: How is this @count counted?

File: app/assets/stylesheets/layout/user/_header.scss [hunk 6]
```diff
@@ -9,10 +9,10 @@
     width: 100%; 
     z-index: 1;
     &__img{
+      font-size: 14px;
       height: 50.46px;
-      width: 140px;
       margin: 7px;
-      font-size: 14px;
+      width: 140px;
```
Comment (ericgpks) [inline app/assets/stylesheets/layout/user/_header.scss]: 💯

**Pull request merged**